src/Datadog.Trace.ClrProfiler.Native/logging.cpp

```diff
@@ -63,8 +63,17 @@ Logger::Logger() {
 
   spdlog::flush_every(std::chrono::seconds(3));
 
+  static auto current_process_name = ToString(GetCurrentProcessName());
+  static auto current_process_id= GetPID();
+  static auto current_process_without_extension =
+      current_process_name.substr(0, current_process_name.find_last_of("."));
```

Comment:
Wondering if this should be the following instead?

```cpp
static auto current_process_without_extension =
    std::filesystem::path(current_process_name).stem();
```

That handles edge cases like the first letter being a `.`, but that's not realistically going to be a problem for us right? WDYT @tonyredondo

Reply:
Well this is defined in C++17 and the linux profiler is compiled using C++11 + some 17 extensions. We can check if that compiles and behave correctly in non windows platforms. If that is the case you have my vote to use it.

Reply:
ah, oops, good point. Didn't notice the usages of `std::filesystem` were in an `#ifdef`. This should be fine then I guess